pkg/services/store/testdata/public_testdata.golden.jsonc

```diff
@@ -12,14 +12,14 @@
 //  }
 //  Name: 
 //  Dimensions: 3 Fields by 2 Rows
-//  +--------------------+----------------------+---------------+
```

Comment:
@matyax do you know what is this?

Reply:
The build was failing, ran the Go tests and this change was made. What is this file? I have no idea.

Reply:
Try merging in latest main and reverting this change out and running the tests again, it should go away?

Reply:
👌  will try.